EventsExpress/Controllers/FrontConfigsController.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace EventsExpress.Controllers
+{
+    using EventsExpress.ViewModels;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Options;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FrontConfigsController : Controller
+    {
+        private readonly IOptions<FrontConfigsViewModel> _frontConfigs;
+
+        public FrontConfigsController(IOptions<FrontConfigsViewModel> frontConfigs)
+        {
+            _frontConfigs = frontConfigs;
+        }
+
+        public IActionResult GetConfigs()
+        {
+            _ = new FrontConfigsViewModel();
```

Comment:
Why do you need this discard value? Remove please.

Reply:
Deleted!